packages/account/src/Components/api-token/__tests__/api-token-table.spec.tsx

```diff
@@ -1,10 +1,16 @@
 import React from 'react';
 import { screen, render } from '@testing-library/react';
+import { useDevice } from '@deriv-com/ui';
 import ApiTokenContext from '../api-token-context';
 import ApiTokenTable from '../api-token-table';
 import { StoreProvider, mockStore } from '@deriv/stores';
```

Comment:
@utkarsha-deriv  we dont have any dependency with stores in the packages/account/src/Components/api-token/api-token-table.tsx component. cant we remove these StoreProvider, mockStore from spec files?

Reply:
Let me check this 🙏